library/build.go

```diff
@@ -919,6 +940,22 @@ func (b *Build) SetDistribution(v string) {
 	b.Distribution = &v
 }
 
+func (b *Build) SetPusherName(v string) {
+	if b == nil {
+		return
+	}
+
+	b.PusherName = &v
+}
```

Comment:
Looks like comments are missing here.

To show an example of what those comments may look like:

https://github.com/go-vela/types/blob/911ffbea2e427883ccca2cc3f3769fdae75e0946/library/build.go#L792-L803